packages/svelte/scripts/check-treeshakeability.js

```diff
@@ -3,6 +3,33 @@ import path from 'node:path';
 import { rollup } from 'rollup';
 import virtual from '@rollup/plugin-virtual';
 import { nodeResolve } from '@rollup/plugin-node-resolve';
+import { compile } from 'svelte/compiler';
+
+async function bundle_code(entry) {
+	const bundle = await rollup({
+		input: '__entry__',
+		plugins: [
+			virtual({
+				__entry__: entry
+			}),
+			nodeResolve({
+				exportConditions: ['production', 'import', 'browser', 'default']
+			})
+		],
+		onwarn: (warning, handle) => {
+			// if (warning.code !== 'EMPTY_BUNDLE') handle(warning);
+		}
+	});
+
+	const { output } = await bundle.generate({});
+
+	if (output.length > 1) {
+		throw new Error('errr what');
```

Comment:
```suggestion
		throw new Error(`Unexpectedly received ${output.length} outputs when only expecting 1.`);
```

Reply:
it's fine as it is, this is just a sanity check for ourselves. a longer message won't make tracking down a bug in the script any easier